client/index.ejs

```diff
@@ -19,6 +19,7 @@
 		<div id="root"></div>
 		<noscript>
 			<p>
+			    This tool requires JavaScript to function. If you do not want to enable Javascript, you will continue to be unable to use this tool.
```

Comment:
That second sentence feels unnecessary. Otherwise this is good. :)

Reply:
Cool. I removed that sentence.